### PR TITLE
Update changelog/package.json for 3.13.1 release

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.13.1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.13.1)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 3.13.1 (2020-04-23)
+
+- Fixed the toggle switch styles to position native inputs correctly within `overflow` containers. [#114](https://github.com/blackbaud/skyux-theme/pull/114)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md:3]: Is `toggle switch styles` correct? The PR we link to shows changes to `sky-switch-input`, but it also links to an that is all about radio buttons, not toggle switches. Just making sure I'm not getting my wires crossed here.

Also, it's minor, but I'd move `correctly` right before `position`. I think this reads more naturally that way, but no big deal if you disagree.

Comment (Blackbaud-SteveBrush) [reply CHANGELOG.md]: Done!

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.13.1](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-release-3.13.1)

**Pull request merged**